ui/main/src/app/services/userSettings/UserSettingsService.ts

```
@@ -8,26 +8,26 @@
 
-export class SettingsService {
+export class UserSettingsService {
     private static userId: string;
-    private static settingsServer;
+    private static userSettingsServer: UserSettingsServer;
 
-    public static setSettingsServer(settingsServer: SettingsServer) {
-        CurrentUserStore.getCurrentUserLogin().subscribe((id) => (SettingsService.userId = id));
-        SettingsService.settingsServer = settingsServer;
+    public static setUserSettingsServer(settingsServer: UserSettingsServer) {
+        CurrentUserStore.getCurrentUserLogin().subscribe((id) => (UserSettingsService.userId = id));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)